src/output_influx.c

```diff
@@ -89,11 +94,11 @@ static void influx_client_event(struct mg_connection *nc, int ev, void *ev_data)
     }
 }
 
-static influx_client_t *influx_client_init(influx_client_t *ctx, char const *url, char const *token)
+static influx_client_t *influx_client_init(influx_client_t *ctx, char const *url, char const *token, const char *metric_format)
 {
-    strncpy(ctx->url, url, sizeof(ctx->url));
-    ctx->url[sizeof(ctx->url) - 1] = '\0';
+    strncpy(ctx->url, url, RTL433_INFLUX_URL_LENGTH);
```

Comment:
I'm not sure about getting rid of the zero termination. We kind of know that the influx_client_t should be zero filled. But glancing at a strncpy without accompanying termination it looks like a bug.